library/built-in/SDXL/_cushySDXLLayout.tsx

```diff
@@ -37,40 +41,74 @@ export function _cushySDXLLayout(): Maybe<DisplaySlotFn<$CushySDXLUI['$Field']>>
       const latent = ui.field.Latent
       ui.set<Field_list<X.XOptional<X.XPrompt>>>('@list.@optional.@prompt^^', {
          Header: false,
-         Body: observer((p) => (
-            <UY.list.BlenderLike<typeof p.field> //
-               field={p.field}
-               renderItem={(item, index) => {
-                  const conditioningIcon: IconName = index == 0 ? 'mdiArrowDown' : 'mdiFormatListGroupPlus'
-                  return (
-                     <UY.Misc.Frame tw='flex items-center' hover key={item.id}>
-                        <span tw={['line-clamp-1 w-full flex-grow px-1', !item.active && 'opacity-50']}>
-                           {item.child.text}
-                        </span>
-                        <div tw='flex-none'>
-                           <IkonOf name={conditioningIcon} />
-                        </div>
-                        <div tw='w-2' />
-                        <div tw='flex-none'>
-                           {/* <InputNumberUI
+         Body: observer((p) => {
+            const positive = ui.field.value.positive
+            const activePrompt = positive.prompts[positive.activeIndex]
+            return (
+               <>
+                  <UY.list.BlenderLike<typeof p.field> //
+                     activeIndex={positive.activeIndex}
```

Comment:
we could pass a persistenceID instead (we would pass the draft id for instance) and the widget would use some local-storage backed value 🤔